doc/modules/model_evaluation.rst

```diff
@@ -215,6 +215,25 @@ the following two rules:
   Again, by convention higher numbers are better, so if your scorer
   returns loss, that value should be negated.
 
+.. note:: **Using custom scorers in functions where `n_jobs > 1`**
```

Comment:
remove backticks aroung n_jobs > 1 as the rendering does not take them into account, see [rendered doc](https://43321-843222-gh.circle-artifacts.com/0/doc/modules/model_evaluation.html#implementing-your-own-scoring-object)

Reply:
Thanks @albertcthomas, for taking the time! :)
I saw those and thought they were a local setup problem.